doc/kernel/usermode/syscalls.rst

```diff
@@ -0,0 +1,259 @@
+.. _syscalls:
+
+System Calls
+############
+
+User threads run with a reduced set of privileges. Certain CPU instructions
```

Comment:
How about:

     User threads run with a reduced set of privileges than Supervisor threads: certain
     CPU instructions may not be used, and they have access to only a limited part
     of the memory map.   System calls (may) allow user threads to perform operations
     not directly available to them.